pom.xml

```diff
@@ -86,9 +82,6 @@
         <bc.version>1.72</bc.version>
         <swagger-annotations-jakarta.version>2.2.8</swagger-annotations-jakarta.version>
 
-        <maven-deploy-plugin.version>3.0.0</maven-deploy-plugin.version>
-        <maven-enforcer-plugin.version>3.2.1</maven-enforcer-plugin.version>
-        <maven-jar-plugin.version>3.3.0</maven-jar-plugin.version>
         <maven-javadoc-plugin.version>3.5.0</maven-javadoc-plugin.version>
```

Comment:
The parent uses `javadoc-plugin` version **3.4.1**, maybe we can downgrade and let it be on Spring Boot.